src/views/checkups/network/details/CheckupsNetworkDetailsPage.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import Loading from '@kubevirt-utils/components/Loading/Loading';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { ResourceYAMLEditor } from '@openshift-console/dynamic-plugin-sdk';
+import { Bullseye, Divider, PageSection, Tab, Tabs, TabTitleText } from '@patternfly/react-core';
+
+import useCheckupsNetworkData from '../hooks/useCheckupsNetworkData';
+import { getJobByName } from '../utils/utils';
+
+import CheckupsNetworkDetailsPageHeader from './CheckupsNetworkDetailsPageHeader';
+import CheckupsNetworkDetailsPageHistory from './CheckupsNetworkDetailsPageHistory';
+import CheckupsNetworkDetailsPageSection from './CheckupsNetworkDetailsPageSection';
+
+import './checkups-network-details-page.scss';
+
+const CheckupsNetworkDetailsPage = () => {
+  const { vmName } = useParams<{ vmName: string }>();
+  const { t } = useKubevirtTranslation();
+  const { configMaps, error, jobs, loading } = useCheckupsNetworkData();
+  const [activeTabKey, setActiveTabKey] = useState<number>(0);
+
+  const configMap = configMaps.find((cm) => cm.metadata.name === vmName);
+  const jobMatches = getJobByName(jobs, configMap?.metadata?.name);
+
+  if (!configMap)
+    return (
+      <Bullseye>
```

Comment:
This is used a lot in the codebase. I suggest we make it a dedicated component.

Reply:
sure

Reply:
On second thought, I've checked the appearances of this bullseye loading, and I see it has about 20 appearances in different files, you're right we should change it, but not in this pr which is big enough and I don't want to add 20 more files to it, should be in separate pr